backend/common/src/main/java/org/eclipse/sw360/mail/MailUtil.java

```diff
@@ -19,6 +19,7 @@
 import org.eclipse.sw360.datahandler.common.SW360Utils;
 import org.eclipse.sw360.datahandler.thrift.ClearingRequestEmailTemplate;
 import org.eclipse.sw360.datahandler.thrift.ThriftClients;
+import org.eclipse.sw360.datahandler.thrift.moderation.ModerationRequest; // Added import
```

Comment:
Please remove unnecessary comments.
```suggestion
import org.eclipse.sw360.datahandler.thrift.moderation.ModerationRequest;
```